data/net/minecraft/commands/arguments/AngleArgument.mapping

```diff
@@ -2,6 +2,7 @@ CLASS net/minecraft/commands/arguments/AngleArgument
 	METHOD getAngle (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)F
 		ARG 0 context
 		ARG 1 name
+	METHOD parse (Lcom/mojang/brigadier/StringReader;)Ljava/lang/Object;
```

Comment:
Please revert this change and the others similar to it in the same package, as this adds nothing. (This will likely be removed by a `sanitizeData` pass.)